Sources/SwiftParser/IncrementalParseTransition.swift

```diff
@@ -109,7 +134,7 @@ public struct IncrementalParseLookup {
     let node = cursorLookup(prevPosition: prevPosition, kind: kind)
     if let delegate = reusedDelegate, let node {
       delegate.parserReusedNode(
-        range: ByteSourceRange(offset: newOffset, length: node.byteSize),
+        range: ByteSourceRange(offset: node.positionAfterSkippingLeadingTrivia.utf8Offset, length: node.byteSizeAfterTrimmingTrivia),
```

Comment:
Maybe the reuse delegate can just receive the reused node instead of the reused range now that everything is implemented in Swift.

Reply:
Yeah, I've made this change.